### PR TITLE
zh: /help/ops/security/root-transition/index.md

### PR DESCRIPTION
**Pull request by fleeto**

### CODE REVIEW

**Comment by fleeto** [comment]
`[trans-bot:pushed]`

Thank you @fleeto, the [related task issue](https://github.com/servicemesher/istio-official-translation/issues/1016) had been updated.

**Comment by loverto** [comment]
- ./public/zh/help/ops/security/root-transition/index.html
  *  internally linking to /zh/docs/setup/kubernetes/upgrade/steps/, which does not exist (line 2285)
     <a href="/zh/docs/setup/kubernetes/upgrade/steps/">Istio 升级指南</a>

File: content_zh/help/ops/security/root-transition/index.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    Checking namespace: default
+      Secret default.istio.default is updated.
+      Secret default.istio.sleep is updated.
+    Checking namespace: istio-system
+      Secret istio-system.istio.default is updated.
+      ...
+    ------All Istio keys and certificates are updated in secret!
+    {{< /text >}}
+
+    If this command fails, wait a minute and run the command again.
```
Comment (loverto) [inline content_zh/help/ops/security/root-transition/index.md]: 英文原文

File: content_zh/help/ops/security/root-transition/index.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+---
+title: 延长自签发证书的有效期
+description: 学习如何延长 Istio 自签发根证书的有效期。
+weight: 90
+keywords: [security, PKI, certificate, Citadel]
+---
+
+因为历史原因，Istio 的自签发证书只有一年的有效期。如果你是用的是 Istio 的自签发证书，就需要在它们过期之前订好计划进行根证书的更迭。根证书过期可能会导致集群范围内的意外中断。
```
Comment (SataQiu) [inline content_zh/help/ops/security/root-transition/index.md]:
```suggestion
因为历史原因，Istio 的自签发证书只有一年的有效期。如果你用的是 Istio 的自签发证书，就需要在它们过期之前订好计划进行根证书的更迭。根证书过期可能会导致集群范围内的意外中断。
```

File: content_zh/help/ops/security/root-transition/index.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+      Secret default.istio.default is updated.
+      Secret default.istio.sleep is updated.
+    Checking namespace: istio-system
+      Secret istio-system.istio.default is updated.
+      ...
+    ------All Istio keys and certificates are updated in secret!
+    {{< /text >}}
+
+    If this command fails, wait a minute and run the command again.
+    It takes some time for Citadel to propagate the certificates.
```
Comment (SataQiu) [inline content_zh/help/ops/security/root-transition/index.md]: 遗漏处，未翻译？

File: content_zh/help/ops/security/root-transition/index.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+    {{< warning >}}
+    这一步骤能够确保控制平面组件和 Envoy Sidecar 全部载入新的证书和密钥
+    {{< /warning >}}
+
+    将控制平面和 `istio-proxy` 升级到 1.0.8、1.1.8 或更新的版本。请依照 [Istio 更新过程](/docs/setup/kubernetes/upgrade/steps/)的介绍来完成这一步骤。
+
+1. 检查 Envoy 是否已经载入新的工作负载证书：
+
+    可以检查一下，Envoy 是否收到了新的证书。要检查检查命名空间 `bar` 之中一个名为 `foo` 的 Pod，验证它的 Envoy 证书的过程：
```
Comment (SataQiu) [inline content_zh/help/ops/security/root-transition/index.md]: 是故意写成 “检查检查”，还是手误重复？
```suggestion
    可以检查一下，Envoy 是否收到了新的证书。要检查命名空间 `bar` 之中一个名为 `foo` 的 Pod，验证它的 Envoy 证书的过程：
```

File: content_zh/help/ops/security/root-transition/index.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+如果下面的命令中显示 `1/1`，说明 Pilot 没有 Sidecar；如果是 `2/2`，就表明 Pilot 是带有 Sidecar 的。
+
+    {{< text bash>}}
+    $ kubectl get po -l istio=pilot -n istio-system
+    istio-pilot-569bc6d9c-tfwjr   1/1     Running   0          11m
+    {{< /text >}}
+
+### 无法使用 sidecar-injector 部署新的工作负载
+
+如果没有更新到 Istio 1.0.8、1.1.8 或者更新的版本，就有可能出现这种情况。请试着重启动 Sidecar injector，重启之后就会载入证书：
```
Comment (SataQiu) [inline content_zh/help/ops/security/root-transition/index.md]:
```suggestion
如果没有更新到 Istio 1.0.8、1.1.8 或者更新的版本，就有可能出现这种情况。请试着重新启动 Sidecar injector，重启之后就会载入证书：
```

File: content_zh/blog/2019/root-transition/index.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+keywords: [security, PKI, certificate, Citadel]
+---
+
+Istio 的自签发证书只有一年的有效期。如果你选择使用 Istio 的自签发证书，就需要在它们过期之前订好计划进行根证书的更迭。根证书过期可能会导致集群范围内的意外中断。这一问题的影响范围涵盖了 1.0.7、1.1.7 及之前的所有版本。
+
+阅读[《延长自签发证书的有效期》](/zh/help/ops/security/root-transition/)一文，其中包含了获取证书有效期以及完成证书轮转的方法。
+
+{{< tip >}}
+
+我们认为每年更换根证书和密钥是一个安全方面的最佳实践，我们会在后续内容中介绍根证书和密钥的翻转方法。
```
Comment (rootsongjc) [inline content_zh/blog/2019/root-transition/index.md]: 根证书和秘钥的“翻转”？不是轮换吗？

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @fleeto, I can only process the PR with 1 file included, will not be reported to the task issues.

**Pull request merged**